presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1136,3 +1136,42 @@ In the following query, the expression CURRENT_TIMESTAMP returns the current tim
             20 | canada        |         2 | comment
             30 | mexico        |         3 | comment
     (3 rows)
+
+Sorted Tables
+^^^^^^^^^^^^^
+
+The Iceberg connector supports to creation of sorted files.
```

Comment:
```suggestion
The Iceberg connector supports the creation of sorted files.
```
I had suggested "The Iceberg connector supports the creation of sorted tables." yesterday. Should this sentence end in "files" or "tables"?

Reply:
Done